study.md

```diff
@@ -66,15 +71,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+no because they are defined separately
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+yes because the_sun is defined with the nearest_star, they have the same value
```

Comment:
Specifically, it's because both identifiers reference the same object in memory.